pages/linux/hashcat.md

```diff
@@ -0,0 +1,32 @@
+# hashcat
+
+> Fast and advanced password recovery tool.
+> More information: <https://hashcat.net/hashcat/>.
+
+- Brute-force attack with default hashcat mask:
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 3 {{hash}}`
```

Comment:
```suggestion
`hashcat --hash-type {{id_hash_type}} --attack-mode {{3}} {{hash}}`
```
Same for all others

Reply:
https://github.com/tldr-pages/tldr/pull/5900#discussion_r627803232